excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/service/ImportServiceImpl.java

```diff
@@ -249,6 +234,17 @@ public DefinitionFileUploadMetadata importFormDefinitions(InputStream inputStrea
 
         metadata.setUserId(userDetails.getEmail());
 
+        // ChallengeQuestionTab
```

Comment:
We probably need to either version challenge questions or on import first delete existing ones for each case type?

Reply:
This is not the agreed requirement in the ticket.  there is a field  id per questions and it create a new entry or update existing one.